### DISCUSSION TITLE
OpenBB Terminal v3.2.2

### DISCUSSION THREAD

**Discussion by jmaslek** [OpenBB-finance/OpenBB#5390] [General]
## Thank you and welcome to our new contributors 🔥
@Define101 and @kulbinderdio

## What's new 🎉

We are excited to launch an implementation of TimeGPT from nixtla!

As per usual, we are also including some general bug fixes, enhancements and documentations.  We have also deprecated some crypto functions that have not been returning data from their API.

## What's changed 🚀
* update defillama_dapps json file (#5380) @Define101
* Allow to execute reports url from reports menu (#5385) @DidierRLopes
* Feature/companies house (#4721) @kulbinderdio
* Hotfix - PyWry fix python 3.11 fail on boot, copy+paste on MacOS (#5384) @tehcoderer
* Change bootup message (#5378) @jmaslek
* Add example notebook for building volume-at-price chart. (#5370) @deeleeramone
* Fix foreach on routine where the inputs provided are not valid (#5373) @DidierRLopes
* 1st integration with TimeGPT-1 (Beta), from Nixtla (#5292) @DidierRLopes
* Removed prints (#5367) @colin99d
* Some error handling (#5357) @colin99d
* Improves the `settings/userdata` command, and upgrade ruff (#5359) @colin99d
* bye bye shroom (#5351) @colin99d
* hotfix / Allow routine args inside lists (#5353) @tehcoderer
* Fix the CPI CSV and remove twitter functionalities (#5350) @jmaslek
* Thegraph totally changed, I removed the endpoints (#5349) @colin99d
* Adds example notebook for installing in Google Colab (#5333) @deeleeramone
* Improve documentation (#5345) @colin99d
* Fixes #5332 (#5344) @colin99d
* hotfix/fix-stocks-search-sdk3: Fixes the stocks search so that it doesn't `print_rich_table()` for SDK. (#5329) @deeleeramone
* Fixed URL for uploading to the hub (#5340) @colin99d
* Fixed the request function (#5336) @colin99d
* Release/3.2.1 (#5335) @jmaslek

We are proud of our community contributors and staunch supporters of open-source ecosystems.
Help us promote our community by tagging `@openbb_finance` on Twitter with a link to your pull request,
and join our Discord server to chat about your contribution! We want to hear about your experience!

### Links 🦋
[Website](https://openbb.co/), [Twitter](https://twitter.com/openbb_finance), [Linkedin](https://www.linkedin.com/company/openbb-finance), [Instagram](https://www.instagram.com/openbb.finance/), [Reddit](https://www.reddit.com/r/openbb/), [Discord](https://discord.com/invite/xPHTuHCmuV)

<hr /><em>This discussion was created from the release <a href='https://github.com/OpenBB-finance/OpenBBTerminal/releases/tag/v3.2.2'>OpenBB Terminal v3.2.2</a>.</em>

**Comment by yasurfer** [comment]
How can i Uninstall or remove openBB?
